src/clojure_ttt/ui.clj

```diff
@@ -55,17 +55,42 @@
        (some (partial = move))
        (boolean)))
 
-(defn input-move [board]
-  (let [s (input)]
-    (cond
+(defn input-move
+  ([board]
+   (input-move board 0))
+
+  ([board depth]
+   (cond
+     (= depth 0) (output "Please choose a move")
```

Comment:
I find name/type of "depth" a bit confusing. We never really care about how far we go down, only that it is not the first one.

I think it could be restructured slightly to avoid this entirely:

input-move
    [board]
    (output "Please choose a move")
    (get-valid-input board)

get-valid-input board
    cond ....
    :else
     (output "Please choose a valid move")
     recur

This captures that we do something only the first time round a bit more explicitly